clusters/hive/pools/stackrox/OWNERS

```diff
@@ -0,0 +1,10 @@
+approvers:
```

Comment:
Maybe add Misha and Marcin to get some EU TZ coverage?

Reply:
done